internals/models.py

```diff
@@ -885,6 +909,48 @@ def getSortingMilestone(feature):
 
     return allowed_feature_list
 
+  @classmethod
+  def get_in_milestone(
+      self, version=None, show_unlisted=False, milestone=None):
+    if milestone == None:
+      return None
+
+    logging.info('Getting chronological feature list in milestone %d', milestone)
+    q = Feature.query()
+    q = q.order(Feature.name)
+    q = q.filter(Feature.shipped_milestone == milestone)
+    desktop_shipping_features = q.fetch(None)
+
+    # Features with an android shipping milestone but no desktop milestone.
+    q = Feature.query()
+    q = q.order(Feature.name)
+    q = q.filter(Feature.shipped_android_milestone == milestone)
+    q = q.filter(Feature.shipped_milestone == None)
+    android_only_shipping_features = q.fetch(None)
+
+    # Constructor the proper ordering.
+    all_features = []
+    all_features.extend(desktop_shipping_features)
+    all_features.extend(android_only_shipping_features)
+
+    # Feature list must be first sorted by implementation status and then by name. 
+    # The implementation may seem to be counter-intuitive using sort() method.
+    all_features.sort(key=lambda f: f.name, reverse=False)
```

Comment:
You could remove reverse=False here and on the next line.

Reply:
Done.